### ISSUE TITLE
export.py

### ISSUE THREAD

**Issue by wuheng0711** [athrunsunny/landmark-face#1]
I tried to export the 'pt' to 'tflite' model by using Yolov5 export.py, but it failed.
How can I modify the code in the export.py?

TensorFlow SavedModel: export failure  2.8s: name 'LmkDetect' is not defined
TensorFlow Lite: starting export with tensorflow 2.10.0...
TensorFlow Lite: export failure  0.0s: 'NoneType' object has no attribute 'call'

**Comment by athrunsunny** [comment]
I updated the code. You can try landmark/export.py

**Comment by wuheng0711** [comment]
Thank you for your update！

1）I can't find the 'yolov5n-lmk.pt' file in the ‘train_landmark.py’.Therefore, I used yolo5n.pt model for training.

2）I used the 'predict.py' file to test the trained model and get the target box and key point information accurately. I think it's right.

3）I use the following statement to export: python ./landmark/export.py --data data/widerhand.yaml --weights runs/train-lmk/exp/weights/best.pt --int8 --include tflite

However, it outputs the following errors, and the model is not successfully converted:
landmark\export: data=E:\98-Yolov5_HandDetect\2-yolov5-master_62\data\widerhand.yaml, weights=E:\98-Yolov5_HandDetect\2-yolov5-master_62\runs\train-lmk\exp\weights\best.pt, imgsz=[192, 192], batch_size=1, device=cpu, half=False, inplace=False, keras=False, optimize=False, int8=True, dynamic=False, simplify=False, opset=11, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['onnx', 'tflite']
YOLOv5  2022-9-30 Python-3.8.13 torch-1.7.1 CPU

Fusing layers... 
YOLOv5n-lmk summary: 157 layers, 1774048 parameters, 0 gradients, 4.2 GFLOPs

PyTorch: starting from E:\98-Yolov5_HandDetect\2-yolov5-master_62\runs\train-lmk\exp\weights\best.pt with output shape (1, 2268, 16) (3.6 MB)

ONNX: starting export with onnx 1.12.0...
ONNX: export failure  0.5s: Exporting the operator silu to ONNX opset version 11 is not supported. Please open a bug to request ONNX export support for the missing operator.

TensorFlow SavedModel: starting export with tensorflow 2.10.0...

                 from  n    params  module                                  arguments                     
2022-10-21 09:04:52.483701: E tensorflow/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2022-10-21 09:04:52.492582: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: ����
2022-10-21 09:04:52.492817: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: ����
2022-10-21 09:04:52.495938: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
  0                -1  1      1760  models.common.Conv                      [3, 16, 6, 2, 2]              
  1                -1  1      4672  models.common.Conv                      [16, 32, 3, 2]                
  2                -1  1      4800  models.common.C3                        [32, 32, 1]                   
  3                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]                
  4                -1  1     29184  models.common.C3                        [64, 64, 2]                   
  5                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]               
  6                -1  1    156928  models.common.C3                        [128, 128, 3]                 
  7                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]              
  8                -1  1    164608  models.common.SPPF                      [256, 256, 5]                 
  9                -1  1    296448  models.common.C3                        [256, 256, 1, False]          
 10                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]              
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 12           [-1, 6]  1         0  models.common.Concat                    [1]                           
 13                -1  1     90880  models.common.C3                        [256, 128, 1, False]          
 14                -1  1      8320  models.common.Conv                      [128, 64, 1, 1]               
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 16           [-1, 4]  1         0  models.common.Concat                    [1]                           
 17                -1  1     22912  models.common.C3                        [128, 64, 1, False]           
 18                -1  1     36992  models.common.Conv                      [64, 64, 3, 2]                
 19          [-1, 14]  1         0  models.common.Concat                    [1]                           
 20                -1  1     74496  models.common.C3                        [128, 128, 1, False]          
 21                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]              
 22          [-1, 10]  1         0  models.common.Concat                    [1]                           
 23                -1  1    296448  models.common.C3                        [256, 256, 1, False]          
TensorFlow SavedModel: export failure  3.1s: name 'LmkDetect' is not defined

TensorFlow Lite: starting export with tensorflow 2.10.0...
TensorFlow Lite: export failure  0.0s: 'NoneType' object has no attribute 'call'

Process finished with exit code 0

**Comment by athrunsunny** [comment]
I know,it need to change the Detect Head in models/tf.py

**Comment by wuheng0711** [comment]
I don't know how to modify models/tf.py. If you can modify it, thank you very much!

**Comment by June1124** [comment]
> I don't know how to modify models/tf.py. If you can modify it, thank you very much!

Have you solved this problem?

**Comment by athrunsunny** [comment]
@wuheng0711 @June1124 I will do it as soon as possible,but it may take some time to modify

**Comment by wuheng0711** [comment]
@athrunsunny  Thank you very much. I will look forward to your update.

**Comment by athrunsunny** [comment]
@wuheng0711 @June1124 This problem has been solved

**Comment by wuheng0711** [comment]
@athrunsunny 
Thank you very much!!!
